builder/builder.go

```diff
@@ -720,3 +871,74 @@ func checkRPM(path string) error {
 	}
 	return nil
 }
+
+func parseUint32(s string) (uint32, error) {
```

Comment:
helpers.go ? The swupd library itself could use this function actually.

Reply:
I'll skip doing this now since the PR is already a long one.